components/MemberDirectory.tsx

```diff
@@ -6,6 +6,9 @@ import Pill from "./Pill";
 
 export interface DirectoryMember extends MemberPublic {
   focus: { active?: boolean; id: string; name: string }[];
+  industry: { active?: boolean; id: string; name: string }[];
+  experienceFilter: { active?: boolean; id: string; name: string }[];
```

Comment:
Hmmm yea this is actually an implementation design from the way we consume new user submissions, notifying @taylorkmho maybe we can discuss this in the future. I'm assuming since this site has been up, a few users already need to shift their years of experience up (assuming they didn't send an update request)

Reply:
I added the issue here: https://github.com/hawaiiansintech/hawaiiansintech/issues/27. It may be simple enough to just keep it as is and have people manually update their profile each year.